src/App.tsx

```diff
@@ -1,25 +1,59 @@
+import { useState, useEffect } from "react";
 import "./App.css";
 import About from "./components/organisms/About/About";
+import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
-import structure from "./structure.json";
 
 const App = () => {
-  const about = structure.about;
-  const gallery = structure.gallery;
+  type JSONData = {
+    about: any
```

Comment:
please create a proper interface and don't use any in this case

Reply:
solved